fart.ini

```diff
@@ -0,0 +1,45 @@
+[app:main]
+use = egg:fart
+
+pyramid.reload_templates = true
+pyramid.debug_authorization = false
+pyramid.debug_notfound = false
+pyramid.debug_routematch = false
+pyramid.debug_templates = true
+pyramid.default_locale_name = en
+
+[server:main]
+use = egg:waitress#main
+host = 0.0.0.0
+port = 6543
```

Comment:
Let's talk about what's best in terms of which ports we run on. Multiple ports? One port?